src/utils.py

```diff
@@ -109,23 +111,18 @@ async def download_file_with_pget(remote_path, dest_path, pget_concurrency="10")
         print(f"[stderr]\n{stderr.decode()}")
 
 
-async def download_files_with_pget(remote_path, path, files, pget_concurrency):
-    await asyncio.gather(
-        *(
-            download_file_with_pget(
-                f"{remote_path}/{file}", f"{path}/{file}", str(pget_concurrency)
-            )
-            for file in files
-        )
-    )
-
-    # # Run the bash script for each missing file
-    # process = subprocess.Popen(["./src/download-with-pget.sh", remote_path, path, *files])
-    # process.wait()
+async def download_files_with_pget(
+    remote_path: str, path: str, files: list[str]
+) -> None:
+    download_jobs = "\n".join(f"{remote_path}/{f} {path}/{f}" for f in files)
+    args = ["pget", "multifile", "/dev/shm/job"]
```

Comment:
Do you want to add `-f`? What is populating /dev/shm/job? You have `download_jobs` above, it looks like that should be passed in on STDIN instead of the file /dev/shm/job. Please correct me if I'm mis-reading this.